platforms/chibios/platform.mk

```diff
@@ -306,6 +306,35 @@ ifeq ($(strip $(USE_CHIBIOS_CONTRIB)),yes)
     EXTRAINCDIRS += $(PLATFORMINC_CONTRIB) $(HALINC_CONTRIB) $(CHIBIOS_CONTRIB)/os/various
 endif
 
+#
+# Raspberry Pi Pico SDK Support
```

Comment:
Rather than directly handle pi stuff here, it would be nice to have something a little more generic like,

```
if "mcu family".mk under platforms/chibios/
    include platforms/chibios/"mcu family".mk
```

Reply:
Done, see updated commit 34092ed5c65711c1f4e75eb9960d507f9d0ba000